### PR TITLE
[bitnami/node-exporter] Add tests and publishing using VIB

### PR DESCRIPTION
**Pull request by FraPazGal**
### Description of the change

The main objective of this PR is to publish our Bitnami Node Exporter Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding Cypress tests.
- Increasing the existing test coverage of the asset by adding GOSS tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/FraPazGal/charts/pull/115

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/node-exporter/templates/psp-clusterrole.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 kind: ClusterRole
```
Comment (joancafom) [inline bitnami/node-exporter/templates/psp-clusterrole.yaml:3]: You also need to update the PSP reference in line 20 to use the `namespace` extension

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for the PR @FraPazGal ! I left you a small comment to review

File: bitnami/node-exporter/templates/psp-clusterrole.yaml (hunk 1 above)
Comment (FraPazGal) [reply bitnami/node-exporter/templates/psp-clusterrole.yaml:3]: Great catch! Changed.

**Review by joancafom — APPROVED** [review]
Thanks for your changes! LGTM!

**Pull request merged**